designsafe/apps/api/agave/filemanager/public_search_index.py

```diff
@@ -144,9 +144,13 @@ def to_file(self):
             dict_obj['meta']['piLabel'] = '{last_name}, {first_name}'.format(
                 last_name=pi_user['last_name'], first_name=pi_user['first_name'])
         else:
-            pi_username = get_user_model().objects.get(username=self.project['value']['pi'])
-            dict_obj['meta']['piLabel'] = '{last_name}, {first_name}'.format(
-                last_name=pi_username.last_name, first_name=pi_username.first_name)
+            try:
+                pi_username = get_user_model().objects.get(username=self.project['value']['pi'])
```

Comment:
if you have the `pi` object defined above on line 140, why not use it for all instances of `self.project['value']['pi']`?

Also the `pi_username` object is really a User instance, not a username, so `pi_username` might be `pi_user` instead

Reply:
Good call. I missed the other `pi` references there, and on the `pi_username` yes that makes more sense I'll change that to `pi_user`.